packages/application-shell/src/components/navbar/navbar-skeleton.styles.tsx

```diff
@@ -90,23 +90,13 @@ const ExpandIcon = styled.div`
   background: rgba(255, 255, 255, 0.2);
 `;
 
-const getPlacementSpecificContainerStyles = (props: TMenuItemProps) =>
-  props.placement === 'header'
-    ? css`
-        background: ${designTokens.colorAccent10};
-      `
-    : css`
-        padding: 12px;
-        height: 48px;
-        background: #009987; // TODO: use new design token color-primary-30
-      `;
-
 const MenuItemContainer = styled.div<TMenuItemProps & TNavBarSkeletonProps>`
   display: flex;
   align-items: center;
   justify-content: center;
   gap: 12px;
-  ${getPlacementSpecificContainerStyles}
+  padding: ${(props) => (props.contentWidth === 'wide' ? '12px' : '12px 28px')};
```

Comment:
I think we need to add one more check here.

`padding: ${(props) => (props.contentWidth === 'wide' || !props.isExpanded ? '12px' : '12px 28px')};`

Because the header icon when placeholder is narrow, disappears because of the big horizontal padding.

<img width="74" alt="Screenshot 2023-08-16 at 13 02 32" src="https://github.com/commercetools/merchant-center-application-kit/assets/52276952/cd529391-6c5c-4790-9190-b22f5b394d36">